insights/parsers/netstat.py

```diff
@@ -677,7 +677,8 @@ class SsTULPN(CommandParser):
     def parse_content(self, content):
         # Use headings without spaces and colons
         SSTULPN_TABLE_HEADER = ["Netid  State  Recv-Q  Send-Q  Local-Address-Port Peer-Address-Port  Process"]
-        self.data = parse_delimited_table(SSTULPN_TABLE_HEADER + content[1:])
+        content = [line for line in content if (('UNCONN' in line) or ('LISTEN' in line))]
```

Comment:
Would you mind to change the docstring of the SsTULPN class to explain that it is now parsing the output of the `ss -tupna` command to extract the same data as the original command?  I think it might be confusing to someone that didn't know the history of the parser.

Reply:
Done